src/server/node_manager.c

```diff
@@ -1370,19 +1369,9 @@ vnode_unavailable(struct pbsnode *np, int account_vnode)
 		degraded_time = presv->ri_degraded_time;
 		in_soonest_occr = find_vnode_in_execvnode(resv_nodes, np->nd_name);
 
-		if (retry_time == 0) {
+		if (retry_time == 0)
 			set_resv_retry(presv, determine_resv_retry(presv));
 
-			str_time = ctime(&presv->ri_resv_retry);
-			if (str_time != NULL) {
-				str_time[strlen(str_time) - 1] = '\0';
-				(void) snprintf(log_buffer, sizeof(log_buffer),
-					"An attempt to reconfirm reservation will be made on %s",
```

Comment:
I don't think anything has changed.  In set_resv_retry() we check if presv->ri_resv_retry is set, and if it isn't, we print 'An attempt'.  That's the same thing that is going on here.

Reply:
But here, it used to check presv->ri_resv_retry and it was set then it used to print "An attempt". In your change "An attempt" is printed when "presv->ri_resv_retry" is not set.

Reply:
I think things were just inconsistent in the code.  IMHO, we should print 'An attempt' the first time, and then 'Next attempt' after that.  This might slightly change some log messages, but I don't think it is bad.